docs/src/staking/stake-accounts.md

```diff
@@ -0,0 +1,106 @@
+## Stake Account Structure
+A stake account on Solana can be used to delegate tokens to validators on
+the network to potentially earn rewards for the owner of the stake account.
+Stake accounts are created and managed differently than a traditional wallet
+address, also known as a system account,
+which is only able to send and receive SOL from other accounts on the network.
+
+Stake accounts on Solana also work differently than those of other Proof-of-Stake
+blockchain networks that you may be familiar with.  This document describes the
+high-level structure and functions of a Solana stake account.
+
+#### Account Address
+Each stake account has a unique address which can be used to look up the account
+information in the command line or in any network explorer tools.  However,
+unlike a wallet address in which the holder of the address's keypair controls
+the wallet, the keypair associated with a stake account address does not necessarily have
+any control over the account.  In fact, a keypair or private key may not even
+exist for a stake account's address.
+
+The only time a stake account's address has a keypair file is when [creating
+a stake account using the command line tools](../cli/staking-operations.md#create-a-stake-account),
+a new keypair file is created first only to ensure that the stake account's
+address is new and unique.
+
+#### Understanding Account Authorities
+Each stake account has two signing authorities specified by their respective address,
+each of which is authorized to perform certain operations on the stake account.
+
+The *stake authority* is used to sign transactions for the following operations:
+ - Delegating stake
+ - Deactivating the stake delegation
+ - Splitting the stake account, creating a new stake account with a portion of the
+ funds in the first account
+ - Merging two undelegated stake accounts into one
+ - Setting a new stake authority
+
+The *withdraw authority* signs transactions for the following:
+ - Withdrawing un-delegated stake into a wallet address
+ - Setting a new withdraw authority
+ - Setting a new stake authority
+
+The stake authority and withdraw authority are set when the stake account is
+created, and they can be changed to authorize a new signing address at any time.
+The stake and withdraw authority can be the same address or two different
+addresses.
+
+The withdraw authority keypair holds more control over the account as it is
+needed to liquidate the tokens in the stake account, and can be used to reset
+the stake authority if the stake authority keypair becomes lost or compromised.
+
+Securing the withdraw authority against loss or theft is of utmost importance
+when managing a stake account.
+
+#### Multiple Delegations
+A stake account may only ever be delegated to one validator at a time. All of
```

Comment:
A stake account doesn't delegate; it's used to delegate SOL. How about: "Each stake account may only be used to delegate to one validator at a time."